packages/adblocker/src/filters/network.ts

```diff
@@ -1152,6 +1163,8 @@ export default class NetworkFilter implements IFilter {
       }
     }
 
+    mask >>>= 0;
```

Comment:
Is this still required after the other PR to generalize the use of setBit has been merged?

Reply:
Nope, this now should be removed.

Reply:
Fixed by https://github.com/ghostery/adblocker/pull/4528/commits/09ea7e724836c7e8683a05ef95556da7b97f7e20